conf/rubocop_rails.yml

```diff
@@ -47,6 +47,10 @@ Rails/UnknownEnv:
     - staging
     - production
 
+Rails/SaveBang:
+  Enabled: true
+  AllowImplicitReturn: false
```

Comment:
Do we want to disable autocorrection?  It's marked as unsafe.  It's not obvious to me what it autocorrects - does it replace methods with their bang version or does it save the result to a local var?

Reply:
It does a straight substitution from, e.g., `save` to `save!`. I'm +1 on disabling autocorrection, particularly since I think it helps enforce the understanding and habit.

Reply:
According to the Slack thread, it apparently replaces with the bang equivalent.

Reply:
I believe it autocorrects by replacing methods with their bang version. Disabling autocorrect seems like a good idea to me - let the developer explicitly choose the correct fix, else they may not have a chance to test/handle the error.